src/admin-toolkit-ui/index.tsx

```diff
@@ -92,143 +81,37 @@ function getAdminToolkitComponent(engine: IEngine) {
   return Array.from(engine.getEntitiesWith(AdminTools))[0][1]
 }
 
-async function initSceneDeployment() {
-  if (deploymentCache !== null) return
-
-  const deployment = await getSceneDeployment()
-
-  if (deployment) {
-    deploymentCache = {
-      data: deployment,
-      deployedBy: deployment.deployedBy.toLowerCase(),
-      sceneBasePosition: deployment.metadata.scene.base.split(','),
-    }
-  }
-}
-
-async function initSceneOwners() {
-  if (sceneOwnersCache !== null) return
+export async function fetchSceneAdmins() {
+  const [error, response] = await getSceneAdmins()
 
-  const owners = await getSceneOwners()
+  console.log(JSON.stringify({ error, response }))
```

Comment:
```suggestion
```

Reply:
i move it inside the error if so if something fails we have logs since there is no error page for this one